applications/extractor.md

```diff
@@ -0,0 +1,175 @@
+
+__Extractor__'s mission is to make Web3 a safer place. By integrating our product into VeChain Ecosystem all projects would be able to use more advanced security solutions and protect their assets.
+
+### Project Details
+
+The product is live in production: https://extractor.hacken.io
+
+High Level Architectural Diagram:
+
+<img src="https://github.com/haas-labs/extractor/raw/main/doc/Ext-Architecture.png" width="800">
+
+<br>
+
+Examples of screens (access to Figma with Full design deck can be provided):
+
+<img src="https://github.com/haas-labs/extractor/raw/main/doc/Ext-Screen-1.png" width="800">
+
+<br>
+<img src="https://github.com/haas-labs/extractor/raw/main/doc/Ext-Screen-2.png" width="800">
+
+<br>
+<img src="https://github.com/haas-labs/extractor/raw/main/doc/Ext-Screen-3.png" width="800">
+
+<br>
+<img src="https://github.com/haas-labs/extractor/raw/main/doc/Ext-Screen-4.png" width="800">
+
+<br>
+
+__Brief Product Slidedeck__:
+
+[https://github.com/haas-labs/extractor/blob/main/doc/Extractor-VeChain.pdf](Extractor-VeChain.pdf)
+
+
+### Ecosystem Fit
+
+Having extensive security experience in the EVM ecosystem both with Smart contracts Audits and automated on-chain detection and protection we bring this capability to the VeChain world by integrating with Extractor. 
+
+Our clients do not need to put in any technical effort as the Extractor team is helping with the product activation and monitoring with no code solution. 
+
+We provide complete protection coverage with the possibility of implementing automatic Smart Contract Actions.
+
+
+## Team 
+
+### Team members
+
+The team will comprise of 3 core engineers:
+
+- Team Lead:  Andriy K, (https://github.com/andriyk-hacken)
+
+- 1 Blockchain Ingest/Integration Data Engineer: Andriy U (https://github.com/)
+
+- 1 BackEnd Developer: Gleb K (https://github.com/gkozyryatskyy)
+
+- 1 Front-End developer: Vova G (https://github.com/vgutorov-hacken)
+
+Support roles:
+
+- Product Owner (Denys I)
+
+### Team Website
+
+- https://extractor.hacken.io
+
+### Team's experience
+
+The team has extensive experience building SaaS web2 and web3 infrastructure products. During the last 1.5 years, the team was involved in building BigData data ingestion and analytics for web3 security.
+
+Members of the team have been at the roots of the Layer-1 Protocol Audit service business.
+
+
+### Team Code Repos
+
+The project is currently in private repositories. We will invite to repositories at your request.
+
+### Team LinkedIn Profiles
+
+- Andriy K (https://www.linkedin.com/in/andriy-kashcheyev-a019382/)
+- Andriy U (https://www.linkedin.com/in/andreyusov/)
+- Gleb K (https://www.linkedin.com/in/gleb-kozyryatskyy-51074395/)
+- Vova G (https://www.linkedin.com/in/vlgutv/)
+
+
+## Development Roadmap 
+
+#### Overview
+
```

Comment:
Our policy stipulates that the maximum allocated budget is $30,000. Please review and adjust your budget accordingly. Thank you for your cooperation.

Reply:
Will have to sync with colleagues on this one.